docs/courses/csintro3/functions/comments.md

```diff
@@ -1,9 +1,213 @@
-# Comments for Functions
+# Activity: Function Documentation
 
-give description of standard function commenting
+In this section, ``||functions:functions||`` have been shown to have a wide variety of uses, with many different features.
 
+For those reasons, we will maintain a particular syntax for ``||functions:function||`` comments. This way, anyone who wants to learn about what the function do will have a consistent and easy way in which to identify these features.
```

Comment:
I suggest changing to "... learn about the function will have ..." or "... learn about what the function does will have ...". 
The reason I prefer the first one is because comments are supposed to describe everything about the function. I feel that saying "what the function does" only encourages writing function descriptions and not param or returns descriptions.